test/Formik.test.tsx

```diff
@@ -44,29 +45,36 @@ function Form({
   );
 }
 
-const InitialValues: Values = { name: 'jared' };
-
-function renderFormik<V>(props?: Partial<FormikConfig<V>>) {
+const InitialValues: Values = {
+  name: 'jared',
+  zipcode: 12345,
+};
+
+function renderFormik<V = Values>(
```

Comment:
I had to add an explicit type definition for the return type here, in order for `getProps()` not to be inferred as returning `any`